.github/workflows/ios-automate.yaml

```diff
@@ -0,0 +1,36 @@
+name: Xcode Project Artifact
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: macos-latest
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        
+      - name: Install Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+          
+      - name: Install dependencies
+        run: npm install
+        
+      - name: Build Chrome extension
```

Comment:
There is no easy way to use the built version of the chrome extension that is already being created, correct? If I'm interpreting correctly, this means that we'll be building it twice? Is it smart enough to not rebuild it again just to convert it?